See more

rairohit17 opened the pull request:
## Description
This pull request enhances the verse similarity feature in our application by implementing see-more functionality, allowing users to access a broader range of similar verses. The following changes have been made:



[Screencast from 2024-10-19 04-04-22.webm](https://github.com/user-attachments/assets/bb82764a-9ecc-4845-8fa3-b87f12fce8a2)


I apologize for the delay in submitting this pull request. There were some bugs in the backend that hindered the frontend's ability to communicate effectively with the backend, which required resolution before proceeding with these enhancements.

rairohit17 committed:
see-more-functionality-added

rairohit17 committed:
bug fix

rairohit17 committed:
Merge remote-tracking branch 'upstream/main' into see-more

JustinhSE commented:
How did you even know that scripture would trigger that 😂. But functionality-wise, it looks good.

JustinhSE commented on backend/routes/verseSearch.py:
```diff
@@ -1,7 +1,15 @@
 bp = Blueprint('verseSearch', __name__)
+CORS(bp, resources={r"/api/*": {"origins": "http://localhost:3000"}})
+
+# Enable CORS for the blueprint (you can also apply it to the app)
```
@Namit2111 are we adding CORS to our project?

JustinhSE commented on backend/routes/__pycache__/__init__.cpython-312.pyc:
why are you pushing binary files? @rairohit17

rairohit17 committed:
removed-mistakenly-added-binary-files

rairohit17 commented:
I’ve removed the mistakenly added binary files and cleaned up the branch. The changes should now be ready for review.

Please let me know  @JustinhSE if any further adjustments are needed. Thanks!

Namit2111 commented:
HI @rairohit17 please resolve the conflicts remove all the binary files and remove cors as of now you are using localhost in cors that is unecessary in the production. Solve these issues and create another PR also squash all the commits while making a PR

JustinhSE commented:
After removing bin files and cors, run `run "npm run build` it will show all the build issues. Resolve these as well
